test/e2e/daemon_restart.go

```diff
@@ -303,7 +303,7 @@ var _ = Describe("DaemonRestart", func() {
 	})
 
 	It("Kubelet should not restart containers across restart", func() {
-
+		SkipIfProviderIs("gke")
```

Comment:
this is already here on L308. 

but probably we should fix the check in the BeforeEach, since returning there won't actually cause the test to be skipped.